docs/customize/vocabularies/affiliations.md

```diff
@@ -4,8 +4,40 @@ Affiliations are a specific type of vocabulary, which enables users to find affi
 for their records' creators and contributors. By using a vocabulary the affiliations are
 deduplicated and can use *search as you type* style suggesters.
 
-In order to import the [ROR](https://ror.org) affiliations dataset you will need to add
-the file containing the dataset itself and then enable it in your instance's `app_data/vocabularies.yaml`
+You need to configure your repository to import the [ROR](https://ror.org)
+affiliations dataset. The recommended approach depends on your version if
+InvenioRDM.
+
+!!! info "Loading time"
+    
+    The ROR vocabulary consists of over 100.000 records and with an ingestion
+    speed around 100-200 records/s it usually takes between around 8-15 minutes
+    to load the records.
+    
+    You can follow the progress via the RabbitMQ management interface
+    on [http://127.0.0.1:15672/](http://127.0.0.1:15672/) (guest/guest).
+
+** Version 12 and later **
```

Comment:
we should standardize how we indicate the version since when a feature works. Like `_Introduced in InvenioRDM v12_`